### ISSUE TITLE
LoggerFactory is not a Logback LoggerContext but Logback is on the classpath

### ISSUE THREAD

**Issue by Bryksin** [jav/expo-server-sdk-java#1]
Hi

I'm not sure where is the problem, on my side or with this package, but after building this project and including this artefact as a dependency (not even using the code, just stating dependency `build.gradle`) it causing the whole application fails to start with error:

```
LoggerFactory is not a Logback LoggerContext but Logback is on the classpath
```
After searching for a solution and reading trough official spring boot documentation I solve it by the method mentioned in the [official documentation](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#howto-configure-log4j-for-logging)

```
dependencies {
    compile 'org.springframework.boot:spring-boot-starter-web'
    compile 'org.springframework.boot:spring-boot-starter-log4j2'
}

configurations {
    all {
        exclude group: 'org.springframework.boot', module: 'spring-boot-starter-logging'
    }
}
```
however, it doesn't feel right, as we are using many other dependencies and none of them requires to explicitly exclude anything from other dependencies, nor causes a complete application failure on dependency include.

P.S. also not having this SDK as an artefact on Maven caused us to make a clone of this repo and include own pipeline Jenkinsfile. Would be really nice to have it in Maven

**Comment by jav** [comment]
I have a few patches I intend on submitting later today.

There is a maven artefact for it. I'll uppdate the README later today.

**Comment by jav** [comment]
The README is now updated with dependency info: https://search.maven.org/artifact/io.github.jav/expo-server-sdk

I have pushed version `0.4.9` which has [removed all usages of sl4j and all logging](https://github.com/jav/expo-server-sdk-java/commit/87c2e93e0c51f19916fe5881936c25c90c9c7ff9) (they where mostly debug-printouts anyway).

The `0.4.9` artifact does not seem to be indexed yet, it's not showing up in https://search.maven.org/artifact/io.github.jav/expo-server-sdk . I assume it is a matter of time (minutes? Hours?). 
I will check in again tomorrow to verify that it was uploaded and indexed correctly.
